### ISSUE TITLE
Reopening Issue #8, Invalid YAML in Templates

### ISSUE THREAD

**Issue by NobleUplift** [groepl/Obsidian-Templates#16]
The YAML in your templates is invalid, so much so that templates break upon generation. I'm not sure why issue #8 was closed without fixing this when the fix was clearly stated, and GitHub agrees:

![firefox_2024-05-30_20-14-05](https://github.com/groepl/Obsidian-Templates/assets/4484679/fb4397d7-ded9-49b0-9c0d-a22e8e9c8089)

I'm also not aware of a single country that uses 2024-05-30, 08:08 PM as a valid time format.

In short, to make the YAML valid it needs to be:

```
created: "{{date}}, {{time}}"
modified: "{{date}}, {{time}}"
```

However, the official format of a date in Obsidian is `YYYY-MM-DDTHH:mm:ss`, so your dates should be compatible with the Obsidian standard: https://help.obsidian.md/Editing+and+formatting/Properties

```
created: "{{DATE:YYYY-MM-DDTHH:mm:ss}}"
modified: "{{DATE:YYYY-MM-DDTHH:mm:ss}}"
```

**Comment by groepl** [comment]
Thank you for your hints and explanations. My YAML seems to be valid with:

```
created: {{date}}, {{time}}
modified: {{date}}, {{time}}
```

but also with:
```
created: "{{date}}, {{time}}"
modified: "{{date}}, {{time}}"
```

It makes no difference in execution. Here is my Obsidian source I used for implementation: https://help.obsidian.md/Plugins/Templates#Template+variables

**Comment by groepl** [comment]
All changes available in v9.0.0 - 24.08.2024
